plugin.localstack.js

```diff
@@ -0,0 +1,56 @@
+/** 
+ * Post-synthesis function to configure AWS provider for LocalStack
+ */
+
+const { Aspects } = require("cdktf");
+
+class LocalStackAspect {
+    endpoint = "localhost.localstack.cloud:4566"
+
+    constructor() {
+    }
+  
+    // This method is called on every Construct within the specified scope (resources, data sources, etc.).
+    visit(node) {        
+        if (node.terraformResourceType == "aws_lambda_function") {
+            node.node.scope._env['AWS_ENDPOINT_URL'] = "http://host.docker.internal:4566";
+            node.node.scope._env['AWS_ENDPOINT_URL_DYNAMODB'] = "http://host.docker.internal:4566";
```

Comment:
ENVs are not respected unfortunately - wondering why https://docs.aws.amazon.com/sdkref/latest/guide/feature-ss-endpoints.html#ss-endpoints-precedence